src/main/java/stream/flarebot/flarebot/Events.java

```diff
@@ -60,6 +63,9 @@
     private static final ThreadGroup COMMAND_THREADS = new ThreadGroup("Command Threads");
     private static final ExecutorService CACHED_POOL = Executors.newCachedThreadPool(r ->
             new Thread(COMMAND_THREADS, r, "Command Pool-" + COMMAND_THREADS.activeCount()));
+    public static List<Long> durations = new ArrayList<>();
+    public static Long avgDuration = 0L;
+    private static List<Long> removedByMe = new ArrayList<>();
```

Comment:
Make these CopyOnWriteList's because we need this thread safe.

Reply:
+1

Reply:
Also make these final and avoid the static if you can

Reply:
Final I can understand, they are staying static because we don't have a way to get an instance of Events.